go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bandprotocol/chain/v3
 
-go 1.22.4
+go 1.22.6
 
 toolchain go1.22.10
```

Comment:
Is there any requirement for us to use the new version?

Reply:
yep